app/channel.go

```diff
@@ -314,7 +314,7 @@ func (a *App) GetOrCreateDirectChannel(userId, otherUserId string) (*model.Chann
 			if esInterface != nil && *a.Config().ElasticsearchSettings.EnableIndexing {
 				a.Srv.Go(func() {
 					for _, id := range []string{userId, otherUserId} {
-						if err := a.indexUserFromId(id); err != nil {
```

Comment:
Clearly here I wouldn't reuse the variable from the outside of the gorouting. I suggest here to rename the error variable instead.

Reply:
I corrected this